src/commands/createFunctionApp/FunctionAppCreateStep.ts

```diff
@@ -43,6 +45,18 @@ export class FunctionAppCreateStep extends AzureWizardExecuteStep<IFunctionAppWi
             siteConfig: await this.getNewSiteConfig(context),
             reserved: context.newSiteOS === WebsiteOS.linux  // The secret property - must be set to true to make it a Linux plan. Confirmed by the team who owns this API.
         });
+
+        context.siteClient = new SiteClient(context.site, context);
+        if (!context.siteClient.isLinux) { // not supported on linux
+            try {
+                await enableFileLogging(context.siteClient);
```

Comment:
I think we talked about enabling this by default for slots as well.

Reply:
Could be - I'm gonna let future Eric deal with that